cypress-tests/cypress/commands/apiCommands.js

```diff
@@ -259,6 +261,50 @@ Cypress.Commands.add("addQueryApi", (queryName, query, dataQueryId) => {
   });
 });
 
+Cypress.Commands.add("addQueryApi1", (queryName, query) => {
```

Comment:
Do we need a new function with this name?
seems like we are using same code in apiAddQueryToApp and this,
please use right naming and let me know if we need this function.

Reply:
fixed , used apiAddQueryToApp this commend and removed newly created one